data/java-cheat-sheet.json

```diff
@@ -0,0 +1,190 @@
+          }
+        ]
+      }, {
+        "title": "স্ক্যানার দিয়ে ইনপুট নেয়া",
+        "items": [
+          {
+            "definition": "স্ক্যানার অব্জেক্ট তৈরি",
+            "code": "Scanner key = new Scanner(System.in)"
+          },{
+            "definition": "ইন্টিজার ইনপুট নেয়া",
+            "code": "key.nextInt()"
+          },{
+            "definition": "ডবল ইনপুট নেয়া",
+            "code": "key.nextDouble()"
+          },{
+            "definition": "লং ইনপুট নেয়া",
+            "code": "key.nextLong()"
+          },{
+            "definition": "স্পেসসহ স্ট্রিং ইনপুট নেয়া",
+            "code": "key.nextLine()"
+          },{
+            "definition": "স্পেস ছাড়া স্ট্রিং ইনপুট নেয়া",
+            "code": "key.next()"
+          },{
+            "definition": "ক্যারেক্টার ইনপুট নেয়া",
+            "code": "key.next().charAt(0)"
+          }
+        ]
+      },{
+          "title": "গাণিতিক ফাংশন",
+          "items": [
+            {
+              "definition": "বর্গমূল করার জন্য",
+              "code": "Math.sqrt()"
+            }, {
+              "definition": "সাধারন লগারিদম পাওয়ার জন্য",
+              "code": "Math.log()"
+            }, {
+              "definition": "এক্সপোনেনশিয়াল নাম্বার এর জন্য",
+              "code": "Math.exp()"
+            }, {
+              "definition": "y এর পাওয়ার হিসেবে x কে নেয়ার জন্য",
+              "code": "Math.pow(x,y)"
+            }, {
+              "definition": "x এর নিকটস্থ, x এর তুলনায় বড়মানের পূর্ণসংখ্যার জন্য",
+              "code": "Math.ceil(x)"
+            }, {
+              "definition": "x এর নিকটস্থ, x এর তুলনায় ছোটমানের পূর্ণসংখ্যার জন্য",
+              "code": "Math.floor(x)"
+            }, {
+              "definition": "x এর এবসোলিউট ভ্যালু এর জন্য ",
+              "code": "Math.abs(x)"
+            }, {
+              "definition": "দৈবচয়নে(রান্ডম) নাম্বার জেনারেট করার জন্য",
+              "code": "Math.random()"
+            }
+          ]
+        },{
+          "title": "প্রচলিত বিশেষ ক্যারেক্টার",
+          "items": [
+            {
+              "definition": "নতুন লাইন প্রর্দশন",
+              "code": "/\n/"
+            }, {
+              "definition": "ট্যাব প্রর্দশন",
+              "code": "/\t/"
+            }, {
+              "definition": "পূর্ণসংখ্যা প্রর্দশন",
+              "code": "%d"
+            }, {
+              "definition": "ফ্লোটিং পয়েন্ট সংখ্যা প্রর্দশন",
+              "code": "%f"
+            }, {
+              "definition": "ডাবল পয়েন্ট সংখ্যা প্রর্দশন",
+              "code": "%lf"
+            }, {
+              "definition": "স্ট্রিং প্রর্দশন",
+              "code": "%s"
+            }, {
+              "definition": "ক্যারেক্টার প্রর্দশন",
+              "code": "%c"
+            }
+          ]
+          }, {
+            "title": "স্ট্রিং ক্লাসের মেথড",
+            "items": [
+              {
+                "definition": "স্ট্রিং এর দৈর্ঘ্য",
+                "code": "strine_name.length()"
```

Comment:
`string` not `strine`. Also can we write variable names using `Java` conventions. That way we can implicitly give a hint about `Java` naming conventions

Reply:
Thank you, vai.
Fixed those things.